activesupport/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 *   Add `String#upcase_first` method.
 
-     *Glauco Custódio*
+    *Glauco Custódio*, *bogdanvlviv*
```

Comment:
Yes, bug fixes the features that was already released. In this case this feature was never released.